### PR TITLE
Add warning about meters and band-aid over the unit issue.

### PR DESCRIPTION
**Pull request by carbotaniuman**

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Why change to specific versions?

**Comment by carbotaniuman** [comment]
I did not mean that. Looks like a debugging artifact, will revert tommorow morning.

File: src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java [hunk 1]
```diff
@@ -16,6 +16,28 @@
...
+			return name;
+		}
+
+		static ExportUnit fromString(String s) {
+			switch (s) {
+				case "Always Meters": return METER;
+				case "Same as Project": return SAME;
+				default: throw new IllegalArgumentException();
+			}
+		}
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java]: This is more robust:
```
    public static ExportUnit fromString(String value) {
        if (value == null) {
            return null;
        }
        return Stream.of(InstanceType.values()).filter(e -> e.toString().equals(value)).findFirst().orElseThrow()
```

File: src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java [hunk 2]
```diff
@@ -16,6 +16,28 @@
...
+
+		ExportUnit(String name) {
+			this.name = name;
+		}
+
+		public String getName() {
+			return name;
+		}
+
+		static ExportUnit fromString(String s) {
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java]:
```suggestion
		public static ExportUnit fromString(String s) {
```

File: src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java [hunk 3]
```diff
@@ -213,6 +255,10 @@ public Values(String lengthUnit, double maxVelocity, double maxAcceleration,
 			return PathUnits.getInstance().length(this.lengthUnit);
 		}
 
+		public ExportUnit getExportUnit() {
+			return ExportUnit.fromString(this.exportUnit);
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java]:
```suggestion
			return ExportUnit.fromString(exportUnit);
```

File: src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java (hunk 1 above)
Comment (AustinShalit) [reply src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java]: If you want to get really fancy you can cache the `values()` call.

**Comment by gftabor** [comment]
@AustinShalit when you are satisfied with changes feel free to merge.

**Pull request merged**